Stripe/STPPaymentCardTextField.m

```diff
@@ -74,10 +77,12 @@ - (instancetype)initWithFrame:(CGRect)frame {
 }
 
 - (void)commonInit {
-    
-    self.borderColor = [self.class placeholderGrayColor];
-    self.cornerRadius = 5.0f;
-    self.borderWidth = 1.0f;
+    _borderColor = [self.class placeholderGrayColor];
+    _cornerRadius = 5.0f;
+    _borderWidth = 1.0f;
```

Comment:
We were doing that before, but UIAppearance tracks when setters are called, and won't override properties that have already been customized (hence the issue in #482).